Source/OxyPlot.Wpf/PlotBase.cs

```diff
@@ -341,14 +347,15 @@ public void ShowTracker(TrackerHitResult trackerHitResult)
                 return;
             }
 
-            var tracker = new ContentControl { Template = trackerTemplate };
-
             // ReSharper disable once RedundantNameQualifier
-            if (!object.ReferenceEquals(tracker, this.currentTracker))
+            if (!object.ReferenceEquals(trackerTemplate, this.currentTrackerTemplate))
```

Comment:
is this a breaking change? will old code not using a tracker template still work?

Reply:
I don't believe so - in the cases where the content of the if block is now skipped (previously it was *always* evaluated since a brand new local variable created on the line above is never the same reference as a member variable) we still set a new DataContext on the control (line 363 in the branch), so even OneTime data bindings would be refreshed.

I tested various scenarios to ensure this is the case, but obviously it's impossible to test this against every possible scenario.

Reply:
OK, I will merge this one. Sorry for the delay!